tests/apps/core/test_pages.py

```diff
@@ -70,16 +70,45 @@ def test_pages_i18n_hreflang(self):
     @override_settings(SENTRY_DSN="https://example.com/sentry/dsn")
     @override_settings(RELEASE="9.8.7")
     @override_settings(ENVIRONMENT="test_pages")
+    @override_settings(
+        RICHIE_LMS_BACKENDS=[
+            {
+                "BASE_URL": "https://lms.example.com",
+                "BACKEND": "richie.apps.courses.lms.edx.EdXLMSBackend",
+                "COURSE_REGEX": r"^https://lms\.example\.com/courses/(?P<course_id>.*)/course/?$",
+                "JS_BACKEND": "base",
+                "JS_COURSE_REGEX": r"^https://lms\.example\.com/courses/(.*)/course/?$",
+            }
+        ]
+    )
     def test_page_includes_frontend_context(self):
         """
         Create a page and make sure it includes the frontend context as included
-        in `base.html`.
+        in `base.html`. All characters for use in javascript string should be
+        escaped. (i.e: '"' should be escaped with \u0022 and '\' with \u005C).
```

Comment:
Same here
```suggestion
        escaped. (i.e: '"' should be escaped with \u0022 and '\' with \u005C).
        to prevent an issue when this variable contains unescaped characters for javascript (e.g: '\').
```